packages/dashboard/src/customization/widgets/utils/assignDefaultStyleSettings.ts

```diff
@@ -4,17 +4,23 @@ import type { QueryWidget } from '../types';
 import { assignDefaultColor } from '@iot-app-kit/core-util';
 import { isDefined } from '~/util/isDefined';
 import { IoTSiteWiseDataStreamQuery } from '~/types';
+import { getCurrentAggregationResolution } from './widgetAggregationUtils';
+import { AggregateType } from '@aws-sdk/client-iotsitewise';
 
 type Query = NonNullable<QueryWidget['properties']['queryConfig']['query']>;
 
-const assignDefaultRefId = (
+// Assigns default RefID to each property and defauly aggregations+resolutions to each property
+const assignDefaults = (
   { assets = [], properties = [] }: IoTSiteWiseDataStreamQuery,
+  resAndAggr: { aggregation?: AggregateType; resolution?: string },
   getId: () => string = uuid
 ) => ({
   assets: assets.map(({ properties, ...others }) => ({
     ...others,
     properties: properties.map((propertyQuery) => ({
       ...propertyQuery,
+      resolution: resAndAggr.resolution,
+      aggregationType: resAndAggr.aggregation,
       refId: propertyQuery.refId || getId(),
     })),
   })),
```

Comment:
I believe you need to do the same for the properties field below this to enable unmodeled data streams.